### DISCUSSION TITLE
Chain two prompts

### DISCUSSION THREAD

**Discussion by darkostanimirovic** [langfuse/langfuse#2346] [Support]
I have two prompts where output of one feeds into the other. I'm only interested in running evaluation on the final result from the second prompt, and ignore the first one.

I'm doing this in Python with LangChain integration where I take one output and pass it into the second prompt. I wonder if this is the right way to do it. I would expect to see one trace with easy access to the first prompt and the second prompt, vs. having two separate traces where I go back and forth – I see this as one conversation, not two, and the score should apply to the whole conversation.

What's the best way to go about this? Thanks!

**Comment by marcklingen** [comment]
Hi @darkostanimirovic, if you use Langchain, you can wrap multiple invocations into a single trace: https://langfuse.com/docs/integrations/langchain/tracing#interoperability

Also, if you pipe the output of one prompt into the next via Langchain itself, these should all be traced within a single trace in Langfuse.

**Reply by datagetabstract** [reply]
Hi @marcklingen, thanks for the great solution. We have product ower users (non coding) who want to test a chain of prompts (output of a prompt as input of another). Could we do this chain run experiment in Langfuse GUI? It would be great to have the feature for non-tech users.
